redaxo/src/core/lib/login/password_policy.php

```diff
@@ -7,8 +7,14 @@
  */
 class rex_password_policy
 {
+    /**
+     * @var array<string, array<string, scalar>>
```

Comment:
```suggestion
     * @var array<string, array{min?: int, max?: int}>
```

Hier könnten wir das doch eigentlich so noch genauer definieren, oder? Sollte eigentlich manche Probleme weiter unten lösen.